substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -224,16 +231,225 @@ mod calls {
 	}
 
 	#[test]
-	fn bail_works() {
+	fn bail_fails_if_called_for_account_none() {
 		ExtBuilder::default().build_and_execute(|| {
-			// TODO
+			assert_err!(SignedPallet::bail(RuntimeOrigin::none()), BadOrigin);
 		})
 	}
 
 	#[test]
-	fn force_clear_submission_works() {
+	fn register_and_submit_page_and_bail_prohibitted_in_phase_other_than_signed() {
 		ExtBuilder::default().build_and_execute(|| {
-			// TODO
+			let account_id = 99;
+
+			let phases = vec![
+				Phase::Off,
+				Phase::SignedValidation(1),
+				Phase::Unsigned(1),
+				Phase::Snapshot(0),
+				Phase::Export(1),
+				Phase::Emergency,
+			];
+
+			for phase in phases {
+				set_phase_to(phase);
+
+				assert_err!(
+					SignedPallet::register(RuntimeOrigin::signed(account_id), Default::default()),
+					NotAcceptingSubmissions::<Runtime>,
+				);
+
+				assert_err!(
+					SignedPallet::submit_page(
+						RuntimeOrigin::signed(account_id),
+						0,
+						Some(Default::default())
+					),
+					NotAcceptingSubmissions::<Runtime>,
+				);
+
+				assert_err!(
+					SignedPallet::bail(RuntimeOrigin::signed(account_id)),
+					NotAcceptingSubmissions::<Runtime>,
+				);
+			}
+		})
+	}
+
+	#[test]
+	fn bail_while_having_no_submissions_does_not_modify_balances() {
```

Comment:
Same as above, `assert_noop!()` will simplify the test case, make it more comprehensive and simpler.